web/job_specs_controller.go

```diff
@@ -77,6 +78,12 @@ func (jsc *JobSpecsController) Show(c *gin.Context) {
 	} else if runs, err := jsc.App.Store.JobRunsFor(j.ID); err != nil {
 		c.AbortWithError(500, err)
 	} else {
-		c.JSON(200, presenters.JobSpec{JobSpec: j, Runs: runs})
+		p := presenters.JobSpec{JobSpec: j, Runs: runs}
+		doc, err := jsonapi.MarshalToStruct(p, nil)
+		if err != nil {
```

Comment:
Would be great to avoid this nested `if`, or at least pull the 82-87 into a helper.